README.md

```diff
@@ -36,63 +36,65 @@ Feel free to post questions on the unofficial StarCraft II AI Discord server. [I
 $ git clone --recursive git@github.com:cpp-sc2/blank-bot.git
 $ cd blank-bot
 
-:: Create build directory.
-$ mkdir build
-$ cd build
-
-:: Create Visual Studio project files.
+:: Create Visual Studio project files in the directory "build"
+:: For Visual Studio 2022
+$ cmake -B build -G "Visual Studio 17 2022"
 :: For Visual Studio 2019
-$ cmake ../ -G "Visual Studio 16 2019"
+$ cmake -B build -G "Visual Studio 16 2019"
 :: For Visual Studio 2017
-$ cmake ../ -G "Visual Studio 15 2017 Win64"
+$ cmake -B build -G "Visual Studio 15 2017 Win64"
 
 :: Build the project using Visual Studio.
-$ start BlankBot.sln
+$ start build\BlankBot.sln
 
 :: Launch the bot with the specified path to a SC2 map, e.g.
-$ bin\Debug\BlankBot.exe Ladder2019Season3/AcropolisLE.SC2Map
+$ build\bin\Debug\BlankBot.exe Ladder2019Season3/AcropolisLE.SC2Map
 ```
 
 ### Linux (command line)
 ```bash
 # Get the project.
 $ git clone --recursive git@github.com:cpp-sc2/blank-bot.git && cd blank-bot
 
-# Create build directory.
-$ mkdir build && cd build
-
-# Generate a Makefile.
-# Use 'cmake -DCMAKE_BUILD_TYPE=Debug ../' if debuginfo is needed
-$ cmake ../
+# Generate CMake build tree
+# Options:
```

Comment:
I believe this list of options is unnecessary because we already have detailed description below.
I would prefer to keep it in one place to reduce the maintenance cost.